packages/contracts-governance/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@eth-optimism/contracts-governance",
+  "version": "0.1.0",
+  "author": "Optimism PBC",
+  "license": "MIT",
+  "main": "dist/index",
+  "types": "dist/index",
+  "files": [
+    "dist/**/*.js",
+    "dist/**/*.d.ts",
+    "dist/types/*.ts",
+    "artifacts/**/*.json",
+    "deployments/**/*.json"
+  ],
+  "scripts": {
+    "build": "npx hardhat compile",
+    "test": "npx hardhat test",
```

Comment:
Any reason to use `npx` here but `yarn solhint` below?

Reply:
this was copy pasted from the other repo, can clean up